go.mod

```diff
@@ -49,9 +44,57 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/go-playground/webhooks.v5 v5.13.0
 	gopkg.in/src-d/go-git.v4 v4.13.1
-	gopkg.in/yaml.v2 v2.2.7
-	k8s.io/api v0.0.0-20191121015604-11707872ac1c
-	k8s.io/apimachinery v0.0.0-20191123233150-4c4803ed55e3
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/utils v0.0.0-20191114200735-6ca3b61696b6 // indirect
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.18.0
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.18.0
+	k8s.io/apiserver v0.18.0
+	k8s.io/cli-runtime v0.18.0
+	k8s.io/client-go v0.18.0
+	k8s.io/cloud-provider v0.18.0
+	k8s.io/cluster-bootstrap v0.0.0
+	k8s.io/code-generator v0.18.0
+	k8s.io/component-base v0.18.0
+	k8s.io/cri-api v0.0.0
+	k8s.io/csi-translation-lib v0.18.0
+	k8s.io/kube-aggregator v0.0.0
+	k8s.io/kube-controller-manager v0.0.0
+	k8s.io/kube-proxy v0.0.0
+	k8s.io/kube-scheduler v0.0.0
+	k8s.io/kubectl v0.0.0
+	k8s.io/kubelet v0.0.0
+	k8s.io/kubernetes v1.18.0
+	k8s.io/legacy-cloud-providers v0.0.0
+	k8s.io/metrics v0.18.0
+	k8s.io/sample-apiserver v0.0.0
+)
+
+// k8s.io/kubernetes has a go.mod file that sets the version of the following
+// modules to v0.0.0. This causes go to throw an error. These need to be set
+// to a version for Go to process them. Here they are set to the same
+// revision as the marked version of Kubernetes. When Kubernetes is updated
+// these need to be updated as well.
```

Comment:
Do we have a source of this method? Just for future reference in understanding it?